phpstan.neon

```diff
@@ -1,5 +1,6 @@
 parameters:
     level: 8
+    checkGenericClassInNonGenericObjectType: false
```

Comment:
Why is it needed, maybe better to fix the errors?

Reply:
I added it because PHPStan complains about classes implementing `DataTransformerInterface` and we have this check turned off in similar cases in core modules. But I figured out how to satisfy PHPStan, so I removed this line :)